panoptes_client/project.py

```diff
@@ -127,5 +127,12 @@ def add_workflows(self, workflows):
             'workflows',
         )
 
+    @property
+    def avatar(self):
+        """
+        A dict containing metadata about the user's avatar.
+        """
+        return Project.avatar_get('{}/avatar'.format(self.id))[0]
```

Comment:
Related to my comment above, the reason this worked before was that it was actually calling the `PanoptesObject.http_get` method here which coincidentally made the right request in this case.

`Project.avatar_get` will be undefined so I think we still need to do some refactoring. Probably just changing this to call `ProjectAvatar.avatar_get(...)` will be enough, but if you have more time I'd be tempted to suggest moving the logic from `ProjectAvatar.avatar_get` directly into `Project.avatar` (since it depends on `project.id`, to me it feels more like that code should be in an instance method on `Project` rather than a class method on `ProjectAvatar`). I'm not sure if I've explained that very well, so talk to me on Slack if not!